framework/contracts/account/src/modules.rs

```diff
@@ -65,18 +62,15 @@ pub fn install_modules(
 pub fn _install_modules(
     mut deps: DepsMut,
     modules: Vec<ModuleInstallConfig>,
-    module_factory_address: Addr,
-    version_control_address: Addr,
     funds: Vec<Coin>,
 ) -> AccountResult<(Vec<SubMsg>, Attribute)> {
     let mut installed_modules = Vec::with_capacity(modules.len());
     let mut manager_modules = Vec::with_capacity(modules.len());
     let account_id = ACCOUNT_ID.load(deps.storage)?;
-    let version_control = VersionControlContract::new(version_control_address);
+    let version_control = VersionControlContract::new(deps.api)?;
 
-    let canonical_module_factory = deps
-        .api
-        .addr_canonicalize(module_factory_address.as_str())?;
+    let canonical_module_factory = CanonicalAddr::from(native_addrs::MODULE_FACTORY_ADDR);
```

Comment:
Can we use the object here?

Reply:
We can, but that will add extra `addr_canonicalize`, because we need canonical address of module factory for whitelisting modules